agent/__init__.py

```diff
@@ -311,6 +311,12 @@ def get_ini_log_file():
     return config['DEFAULT'].get('log_file', None)
 
 
+def get_enroll_token():
+    config = configparser.ConfigParser()
+    config.read(INI_PATH)
+    return config['DEFAULT'].get('enroll_token', None)
```

Comment:
`.get` returns None by default. so pls stop reproducing this bad design pattern

Reply:
fixed